exercises/series/src/test/java/SeriesTest.java

```diff
@@ -17,145 +17,117 @@
     public ExpectedException expectedException = ExpectedException.none();
 
     @Test
-    public void hasDigitsShort() {
-        Series series = new Series("01234");
-        List<Integer> expected = Arrays.asList(0, 1, 2, 3, 4);
-        List<Integer> actual = series.getDigits();
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
-        assertEquals(expected, actual);
+    public void slicesOfOneFromOne() {
+		Series series = new Series("1");
+		List<List<Integer>> expected = Arrays.asList(Collections.singletonList(1));
```

Comment:
From the canonical data it looks like the expected return type of `slices` is a list of strings instead of a list of lists of integers 🙂